gammapy/spectrum/spectrum_results.py

```diff
@@ -203,6 +203,20 @@ def from_json(data):
 
         return model
 
+    @staticmethod
+    def from_sherpa(covar):
+        """Read sherpa fit output
+
+        Parameters
+        ----------
+        covar : `~sherpa.fit.ErrorEstResults`
+            Fit results as return by :func:`~sherpa.astro.datastack.covar`
+        """
+        model = SpectralModelPowerLaw()
+        model.meta = Bunch(covar.__dict__)
```

Comment:
I need to look at the code and think about it a bit.
If you know a way how to add it that works for you and want something ASAP, go ahead.
Otherwise I can have a look after lunch.